qualcoder/code_text.py

```diff
@@ -458,6 +528,38 @@ def merge_codes(self, item, parent):
         # update filter for tooltip
         self.eventFilterTT.setCodes(self.code_text, self.codes)
 
+    def add_link(self):
```

Comment:
I am a bit lost here, why are codes called links now? why is there a linetype and what does a linetype do?